smart_proxy_remote_execution_ssh.gemspec

```diff
@@ -14,7 +14,8 @@ Gem::Specification.new do |gem|
     Ssh remote execution provider for Foreman Smart-Proxy
   EOS
 
-  gem.files            = Dir['{bundler.d,lib,settings.d}/**/*', 'LICENSE', 'README.md']
+  gem.files            = Dir['lib/smart_proxy_remote_execution_ssh.rb', 'LICENSE', 'README.md',
+                             '{bundler.d,lib,settings.d}/**/*']
```

Comment:
this should not contain the whole lib but `lib/smart_proxy_remote_execution_ssh/**/*` and `lib/smart_proxy_remote_execution_ssh.rb` only, otherwise smart_proxy_remote_execution_ssh plugin tries to load the core as well